tests/common/__init__.py

```diff
@@ -1,12 +1,22 @@
+import types
+from typing import List, Type
+
 from tests.common.compare import deep_compare
 from tests.common.datasets import (RandomClassificationDataset, RandomClassificationDatasetHparams, RandomImageDataset,
                                    configure_dataset_hparams_for_synthetic)
 from tests.common.events import EventCounterCallback, EventCounterCallbackHparams
+from tests.common.hparams import assert_is_constructable_from_yaml, assert_registry_contains_entry
 from tests.common.markers import device, world_size
 from tests.common.models import (SimpleConvModel, SimpleConvModelHparams, SimpleModel, SimpleModelHparams,
                                  configure_model_hparams_for_synthetic)
 from tests.common.state import assert_state_equivalent
 
+
+def get_all_subclasses_in_module(module: types.ModuleType, cls: Type) -> List[Type]:
```

Comment:
nit: can omit the `all` for brevity, or even better `get_module_subclasses`

Reply:
Done